src/cogs/bert.py

```diff
@@ -128,6 +147,24 @@ async def norbert(self, ctx):
         if ctx.invoked_subcommand is None:
             await ctx.send_help(ctx.command)
 
+    @norbert.command(name="top")
+    async def top(self, ctx, *content):
```

Comment:
method name has to differ from the other bert "top" command to avoid naming conflicts